shared/src/components/exercise/mode.cjsx

```diff
@@ -31,7 +31,7 @@ ExMode = React.createClass
 
   componentDidUpdate: (prevProps) ->
     {mode, focus} = prevProps
-    @setFreeResponseFocusState() if mode is 'free-response' and focus isnt @props.focus
+    @setFreeResponseFocusState() if mode is 'free-response'
```

Comment:
the way this is set up right now, any type of update will scroll to the top, even if im typing in the textarea.  is it that on mount, we need to scroll to the top?  I'm not sure i understand what the card is asking for